Smartsheet.Net.Standard/Entities/CellHistory.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Smartsheet.Net.Standard.Interfaces;
+
+namespace Smartsheet.Net.Standard.Entities
+{
+    public class CellHistory : ISmartsheetObject
```

Comment:
Forgot to add this comment the first time. This implementation should just extend the `Cell` object implementation, following the object definition in the API, rather than reimplementing ColumnId, DisplayValue, and Value. There are also additional properties that can be fetched with include flags that would be dropped here otherwise.

Reply:
Updated, added columnType to Cell object as it was missing.